client/service/src/lib.rs

```diff
@@ -140,6 +141,8 @@ pub struct PartialComponents<Client, Backend, SelectChain, ImportQueue, Transact
 	pub import_queue: ImportQueue,
 	/// A shared transaction pool.
 	pub transaction_pool: Arc<TransactionPool>,
+	/// A shared statement store.
+	pub statement_store: Option<Arc<StatementStore>>,
```

Comment:
```suggestion
```

Can just be passed in `other`.

Reply:
I did not realise that `other` is extensible. Fixed now.